src/v/config/node_config.cc

```diff
@@ -26,108 +26,114 @@ node_config::node_config() noexcept
+ The `node_id` property must not be changed after a broker joins the cluster.",
       {.visibility = visibility::user},
       std::nullopt,
       [](std::optional<model::node_id> id) -> std::optional<ss::sstring> {
-          if (id && (*id)() < 0) {
-              return fmt::format("Negative node_id ({}) not allowed", *id);
-          }
-          return std::nullopt;
+    if (id && (*id)() < 0) {
+        return fmt::format("Negative node_id ({}) not allowed", *id);
+    }
+    return std::nullopt;
       })
   , rack(
       *this,
       "rack",
-      "Rack identifier",
+      "A label that identifies a failure zone. Apply the same label to all brokers in the same failure zone. When `enable_rack_awareness` is set to `true` at the cluster level, the system uses the rack labels to spread partition replicas across different failure zones.",
       {.visibility = visibility::user},
       std::nullopt)
   , seed_servers(
       *this,
       "seed_servers",
-      "List of the seed servers used to join current cluster. If the "
-      "seed_server list is empty the node will be a cluster root and it will "
-      "form a new cluster",
+      "List of the seed servers used to join current cluster. If the `seed_servers` list is empty the node will be a cluster root and it will form a new cluster.
+
+* When `empty_seed_starts_cluster` is `true`, Redpanda enables one broker with an empty `seed_servers` list to initiate a new cluster. The broker with an empty `seed_servers` becomes the cluster root, to which other brokers must connect to join the cluster.  Brokers looking to join the cluster should have their `seed_servers` populated with the cluster root's address, facilitating their connection to the cluster.
+
+
+Only one broker, the designated cluster root, should have an empty `seed_servers` list during the initial cluster bootstrapping. This ensures a single initiation point for cluster formation.
+
+When `empty_seed_starts_cluster` is `false`, Redpanda requires all brokers to start with a known set of brokers listed in `seed_servers`. The `seed_servers` list must not be empty and should be identical across these initial seed brokers, containing the addresses of all seed brokers. Brokers not included in the `seed_servers` list use it to discover and join the cluster, allowing for expansion beyond the foundational members.
+
+
+The `seed_servers` list must be consistent across all seed brokers to prevent cluster fragmentation and ensure stable cluster formation.
+",
       {.visibility = visibility::user},
       {},
       [](std::vector<seed_server> s) -> std::optional<ss::sstring> {
-          std::sort(s.begin(), s.end());
-          const auto s_dupe_i = std::adjacent_find(s.cbegin(), s.cend());
-          if (s_dupe_i != s.cend()) {
-              return fmt::format(
-                "Duplicate items in seed_servers: {}", *s_dupe_i);
-          }
-          return std::nullopt;
+    std::sort(s.begin(), s.end());
```

Comment:
I don't think you meant to change this, looks like a formatter did it.

Reply:
Yes, that's correct.